### ISSUE TITLE
Inconsistent document/webpage link to this repo

### ISSUE THREAD

**Issue by TanNgocDo** [second-state/dapr-wasm#54]
I see there are multiple article link to this repo with obsolete information:
https://www.infoq.com/articles/webassembly-dapr-wasmedge/?utm_source=email&utm_medium=Web-Development&utm_campaign=newsletter&utm_content=11162021
https://wasmedge.org/book/en/use_cases/frameworks/mesh/dapr.html

It seems above links are having a Golang microservice, I think it's better if you can maintain both repo or update again the article.
Thanks

**Comment by juntao** [comment]
Yeah. I am not sure if InfoQ allows us to update the article. But the WssmEdge book chapter definitely need to be updated. 

Cheers
Michael
